Services/OpenIdConnect/classes/class.ilOpenIdConnectSettingsGUI.php

```diff
@@ -57,7 +59,7 @@ public function __construct(int $a_ref_id)
         $this->review = $DIC->rbac()->review();
         $this->error = $DIC['ilErr'];
         $this->upload = $DIC->upload();
-
+        $this->body = $DIC->http()->request()->getParsedBody();
```

Comment:
@pascalseeland IMO this is always an array `$DIC->http()->request()->getParsedBody();`